### DISCUSSION TITLE
Where is the list of supported langauges?

### DISCUSSION THREAD

**Discussion by jcubic** [github-linguist/linguist#6683] [Q&A]
I've added this:

```
Parser.peg.php linguist-language=PEG
Expression.php linguist-generated
```

to the repo [jcubic/expression.php](https://github.com/jcubic/expression.php)

But it has no effect, PEG is ignored. Where can you find the list of supported languages? I can't find it anywhere.

**Comment by lildude** [comment]
All languages are listed in the [`languages.yml`](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml) file.

I'm not sure what PEG is, but we have support for [PEG.js](https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L4981-L4990).
